types/office-js-preview/index.d.ts

```diff
@@ -120227,91 +120143,91 @@ declare namespace Visio {
         interface ApplicationUpdateData {
             /**
-             *
              * Show or hide the iFrame application borders.
              *
+             * @remarks
              * [Api set:  1.1]
              */
             showBorders?: boolean;
             /**
-             *
              * Show or hide the standard toolbars.
              *
+             * @remarks
              * [Api set:  1.1]
              */
             showToolbars?: boolean;
         }
         /** An interface for updating data on the Document object, for use in `document.set({ ... })`. */
         interface DocumentUpdateData {
             /**
-            *
             * Represents a Visio application instance that contains this document.
             *
+            * @remarks
             * [Api set:  1.1]
             */
             application?: Visio.Interfaces.ApplicationUpdateData;
             /**
-            *
             * Returns the DocumentView object.
             *
+            * @remarks
             * [Api set:  1.1]
             */
             view?: Visio.Interfaces.DocumentViewUpdateData;
         }
         /** An interface for updating data on the DocumentView object, for use in `documentView.set({ ... })`. */
         interface DocumentViewUpdateData {
             /**
-             *
              * Disable Hyperlinks.
              *
+             * @remarks
              * [Api set:  1.1]
              */
             disableHyperlinks?: boolean;
             /**
-             *
              * Disable Pan.
              *
+             * @remarks
              * [Api set:  1.1]
              */
             disablePan?: boolean;
             /**
-             *
              * Disable PanZoomWindow.
              *
+             * @remarks
              * [Api set:  1.1]
              */
             disablePanZoomWindow?: boolean;
             /**
-             *
              * Disable Zoom.
              *
+             * @remarks
              * [Api set:  1.1]
              */
             disableZoom?: boolean;
             /**
-             *
              * Hide Diagram Boundary.
              *
+             * @remarks
              * [Api set:  1.1]
              */
             hideDiagramBoundary?: boolean;
         }
         /** An interface for updating data on the Page object, for use in `page.set({ ... })`. */
         interface PageUpdateData {
             /**
-            *
             * Returns the view of the page.
             *
+            * @remarks
             * [Api set:  1.1]
             */
             view?: Visio.Interfaces.PageViewUpdateData;
         }
         /** An interface for updating data on the PageView object, for use in `pageView.set({ ... })`. */
         interface PageViewUpdateData {
             /**
-             *
              * Get and set Page's Zoom level. The value can be between 10 and 400 and denotes the percentage of zoom.
```

Comment:
This and other instances

```suggestion
             * Gets and sets page's zoom level. The value can be between 10 and 400 and denotes the percentage of zoom.
```